Detector/doc/DD4hepInFCCSW.md

```diff
@@ -357,4 +365,32 @@ export LIBGL_ALWAYS_INDIRECT=1
 
 FCCSW folder structure
 --
-Currently all xml-descriptions are in `DetectorDescription/Detectors/compact` and all implementations in `DetectorDescription/Detectors/src`. This will likely change in the near future.
+The detector descriptions and geometry services / tools can be found in the `Detector` package. The FCCSW components that provide the geometry to the remaining packages can be found in:
+- DetInterface (virtual interface classes)
+- DetComponents (actual implementations of the services / tools)
+
+Extensions of the DD4hep description are found in:
+- DetExtensions (for reconstruction)
+- DetSensitive (custom sensitive volumes)
+
+### The master XMLs
+The descriptions are placed in sub-folders of the Detector package. We plan to have the possibility of having several baseline detectors for the different accelerators. For each such baseline we will have a sub-directory `"DetFCC" + Flavour + "Baseline" + X` where X will be an increasing index and `Flavour = ee, eh, hh`, selected to describe the accelerator the baseline is meant for (for example `DetFCChhBaseline1`). In each baseline description we expect to have a `compact` directory with:
```

Comment:
Can you add a little ascii graphic of the directory layout? Should be easier to understand then.